packages/docs/celo-holder-guide/eth-recovery.md

```diff
@@ -46,6 +46,12 @@ This command will return you with:
 - `privateKey`: the private key associated with your address -- please record this private key on paper and not share with anyone else
 - `publicKey`: the public key associated with your address
 
+***Note***
+
+If the address was generated with another derivationPath or address index (the ethereum's default is `"m/44'/60'/0'/0/0"` which will cover almost every case) you can specify both using the flags `addressIndex` and `derivationPath`.
+
+Example for `m/44'/78'/1'/1/23` derivationPath: `celocli account:new --mnemonicPath recovery.txt --derivationPath "m/44'/78'/1'/1" --addressIndex 23 --node https://forno.celo.org`.
```

Comment:
This isn't quite right.  The command as given will use the path `m/44'/78'/1'/1/0/23`, because it defaults to `changeAddress` being 0 (can be verified using https://iancoleman.io/bip39/).  To get `m/44'/78'/1'/1/23` you could use:
 
```
celocli account:new --mnemonicPath recovery.txt --derivationPath "m/44'/78'/1'" --changeAddress 1 --addressIndex 23 --node https://forno.celo.org
```

However, this relates to a difficulty I've run into helping a user recover their funds.  They had used an address generated using a Ledger in 2017, at which point Ledger was using a different derivation path: `m/44'/60'/0'/[addressIndex]` rather than `m/44'/60'/0'/[addressIndex]` like they do now.  Unfortunately, since the changeAddress part of the path in our code isn't optional (if you omit it, it defaults to zero rather than no change address), we can't handle this path using celocli (I had to point the user to https://iancoleman.io/bip39/).  One way to possible resolve this would be to support `--changeAddress null` or something like that, though I don't know what syntax would give the best UX.  I imagine we should file this as a separate issue, but I would certainly like to see it resolved in some way.

Reply:
Or perhaps we could add another special preset `ethLegacyLedger`.  But it would still have to involve some change to `generateKeys()` so it can omit the change portion of the path.  See https://github.com/celo-org/celo-monorepo/blob/2adcf1eeca33910387e0c4e5fde3afda8f035901/packages/sdk/utils/src/account.ts#L167-L184

Reply:
Ok, I'll change the docs for now, and we could add an issue to fix that